WebFeedGatewayPlugin.php

```diff
@@ -126,7 +128,8 @@ public function fetch($args, $request): bool
                     'latestDate' => $latestDate,
                     'feedUrl' => $request->getRequestUrl(),
                     'userGroups' => $userGroups,
-                    'includeIdentifiers' => $includeIdentifiers
+                    'includeIdentifiers' => $includeIdentifiers,
+                    'language' => LocaleConversion::getIso1FromLocale(Locale::getLocale()),
```

Comment:
The `getIso1FromLocale()` will lose the country information, but it's supported by the specification.